dynawo/sources/API/JOB/JOBXmlHandler.cpp

```diff
@@ -380,6 +380,8 @@ InitValuesHandler::create(attributes_type const& attributes) {
   initValuesEntry_ = shared_ptr<InitValuesEntry>(new InitValuesEntry());
   initValuesEntry_->setDumpLocalInitValues(attributes["local"]);
   initValuesEntry_->setDumpGlobalInitValues(attributes["global"]);
+  if (attributes.has("init"))
```

Comment:
we could make the other two attributes optional as well

Reply:
yes, I propose to open an independent ticket

Reply:
fine